pkg/server/handler/traitHandler.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/cloud-native-application/rudrx/pkg/server/apis"
 	"github.com/cloud-native-application/rudrx/pkg/server/util"
 	"github.com/gin-gonic/gin"
+	ctrl "sigs.k8s.io/controller-runtime"
```

Comment:
do we really need to import this lib only for logs？

Reply:
Not sure, but this lib is widely used across all API services.